periodic-table/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: PERIODIC TABLE 
+uuid: periodic-table
+prerequisites:
+  - 
+learning_objectives: 
+  - 
+references:
```

Comment:
This blank item will throw an error. Try "references: None" or just leave references out of the header entirely.